posts.go

```diff
@@ -424,6 +425,18 @@ func handleViewPost(app *App, w http.ResponseWriter, r *http.Request) error {
 		}
 	}
 
+	var protectDraft bool
+	if found && collectionID.Valid {
+		collection, err := app.db.GetCollectionByID(collectionID.Int64)
+		if err != nil {
+			log.Error("view post: %v", err)
+		}
+
+		protectDraft = (collection.IsPrivate() || collection.IsProtected())
+	} else {
+		protectDraft = true
```

Comment:
I believe this can be removed? Seems to make every single draft post private, even when it doesn't belong to a private / protected collection.